monkey/tests/unit_tests/infection_monkey/credential_collectors/test_ssh_credentials_collector.py

```diff
@@ -67,5 +77,7 @@ def test_ssh_info_result_parsing(monkeypatch, patch_telemetry_messenger):
         Credentials(identity=username3, secret=None),
         Credentials(identity=None, secret=ssh_keypair3),
     ]
-    collected = SSHCredentialCollector(patch_telemetry_messenger).collect_credentials()
+    collected = SSHCredentialCollector(
+        patch_telemetry_messenger, mock_event_queue
+    ).collect_credentials()
     assert expected == collected
```

Comment:
Do we have a test that verifies that credentials events get published?

Reply:
I am not really sure how I am going to test that.